apps/movie-quotes-admin/src/App.js

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Admin, Resource, ListGuesser, EditGuesser, ShowGuesser } from 'react-admin';
+import platformaticDbRestProvider from './ra-data-platformatic-db-rest';
+
+const restApiUrl = process.env.REACT_APP_PLATFORMATIC_DB_REST_API_URL;
+const dataProvider = platformaticDbRestProvider(restApiUrl);
+
+const App = () => (
+    <Admin dataProvider={dataProvider}>
+        <Resource name="movies" list={ListGuesser} show={ShowGuesser} edit={EditGuesser} />
+        <Resource name="quotes" list={ListGuesser} show={ShowGuesser} edit={EditGuesser} />
```

Comment:
Can't we compose this list of Resources dynamically from the OpenAPI definitions?

If that's possible, then it would be significantly better to bundle this within https://github.com/platformatic/platformatic/tree/main/packages/db-dashboard.

Reply:
Yes, sure is exactly my idea... I would like implement it inside db-dashboard.
I have manually implemented a react-admin app in this demo to show you how could be simple generate the App.js dinamically using OpenAPI definitions, sure!

Reply:
@mcollina let's move here to discuss about this feature implementation, please:
https://github.com/platformatic/platformatic/issues/33

Reply:
@mcollina 
If you agree, I would close this PR as it is outdated based on this:
https://github.com/platformatic/platformatic/pull/255

Reply:
@ricfio I agree it makes sense to close this PR now — go for it!